### PR TITLE
JavaScript & OAuth bug fixes

### LINKED ISSUE #62: populateAccountList() should run on page load
Right now, the populateAccountList() JS function is executing only after the selected vendor changes, but this causes a bug if the user wants to select an account of the vendor that is selected on page load.

### LINKED ISSUE #87: populateEditForm chooses the wrong account
> It is displaying the contents of the first Account of a config instead of the account that was specified by the user.

### PR DESCRIPTION
**Pull request by cadecrandall**
### Description
This PR addresses some bugs we found in the demo.
- Changes redirect from index to edit file following a successful post request
- Catches NullPointerException thrown when the domain environment variable has not been set and throws better error message
- Populates list of customers and accounts on page load
- Populates the selected account information on edit form, rather than the first in the config

### Issues resolved
- Closes #87 
- Closes #62

### CODE REVIEW

File: src/main/java/servlets/OAuthConsentServlet.java [hunk 1]
```diff
@@ -43,9 +43,13 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     // Get the current session or create a new one.
     HttpSession session = request.getSession(true);
     session.setAttribute(OAuthConstants.SHEETS_SESSION_KEY, state);
-
-    // Redirect user to OAuthConsentPage.
-    response.sendRedirect(getOAuthRedirectURL(state));
+    try {
+      String redirectUrl = getOAuthRedirectURL(state);
+      // Redirect user to OAuthConsentPage.
```
Comment (cxfloeder) [inline src/main/java/servlets/OAuthConsentServlet.java:48]: Probably don't need this comment.

File: src/main/webapp/scripts/appscript.js [hunk 2]
```diff
@@ -78,6 +80,7 @@ function buildEditForm() {
   showForm();
 }
 
+
 /**
  * Builds a query string specific to the edit form to fetch an entire configuration
```
Comment (cxfloeder) [inline src/main/webapp/scripts/appscript.js:85]: Remember periods

File: src/main/java/servlets/BillingConfig.java [hunk 3]
```diff
@@ -29,7 +29,7 @@
 @WebServlet("/BillingConfig")
 public class BillingConfig extends HttpServlet {
   private static final String CONTENT_TYPE_APPLICATION_JSON = "application/json;";
-  private static final String REDIRECT_INDEX = "/index.html";
+  private static final String REDIRECT_INDEX = "/editfile.html";
```
Comment (ianjjohnson) [inline src/main/java/servlets/BillingConfig.java:32]: Rename to REDIRECT_EDIT?

File: src/main/java/servlets/OAuthConsentServlet.java [hunk 4]
```diff
@@ -43,9 +43,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
...
     HttpSession session = request.getSession(true);
     session.setAttribute(OAuthConstants.SHEETS_SESSION_KEY, state);
-
-    // Redirect user to OAuthConsentPage.
-    response.sendRedirect(getOAuthRedirectURL(state));
+    try {
+      String redirectUrl = getOAuthRedirectURL(state);
+      response.sendRedirect(redirectUrl);
+    } catch(NullPointerException e) {
+      response.sendError(400, "The domain variable has not been set");
```
Comment (ianjjohnson) [inline src/main/java/servlets/OAuthConsentServlet.java:50]: Might be nice to have a more user-friendly error message here. This one seems a bit cryptic

**Pull request merged**